lib/plugins/backup.rb

```diff
@@ -45,6 +48,9 @@ def backup(resources) # rubocop:disable Metrics/AbcSize
         @app_name = resources.stack.app_name
         @stack_name = resources.stack.name
         @target_name = render(@target_name)
+        @target_bucket = define_bucket
+
+        return if @target_bucket.nil?
```

Comment:
@askreet at this point, a single bucket string or a hash map is definitely provided, but it's also possible that the hash map doesn't contain the current account as a key. I'd suggest to silence the plugin in this scenario, eg. we don't want to back up the archives during system tests.